pkg/components/power-monitor/deployment.go

```diff
@@ -415,9 +415,15 @@ func newPowerMonitorContainer(pmi *v1alpha1.PowerMonitorInternal) corev1.Contain
 		Name:            pmi.DaemonsetName(),
 		SecurityContext: &corev1.SecurityContext{Privileged: ptr.To(true)},
 		Image:           deployment.Image,
```

Comment:
@vimalk78 this needs the new image - right? Could you please update the image to 0.0.8 ?